models/points.go

```diff
@@ -353,7 +353,7 @@ func (p *pointsAPI) payByPrime(pts PointsToken) (rest PaymentResp, err error) {
 		// 100 would become 1 TWD in TapPay
 		"amount":   0 - pts.Points.Currency,
 		"currency": config.Config.PaymentService.Currency,
-		"details":  fmt.Sprintf("%s %s", payment_desc, pts.CreatedAt),
+		"details":  fmt.Sprintf("%s %v", payment_desc, pts.CreatedAt),
```

Comment:
@FalseChord I have to change these two part to let the make process successful. Or it will show the error messages and fail:

```bash
models/points.go:221:30: Sprintf format %s reads arg #2, but call has 1 arg
models/points.go:356:15: Sprintf format %s has arg pts.CreatedAt of wrong type github.com/readr-media/readr-restful/models.NullTime
make: *** [Makefile:15: test] Error 2
```